stdlib/public/core/KeyPath.swift

```diff
@@ -2507,7 +2507,7 @@ internal enum KeyPathPatternStoredOffset {
   case inline(UInt32)
   case outOfLine(UInt32)
   case unresolvedFieldOffset(UInt32)
-  case unresolvedIndirectOffset(UnsafePointer<UInt32>)
+  case unresolvedIndirectOffset(UnsafePointer<UInt64>)
```

Comment:
This should be `UInt`, for 32-bit portability. LGTM otherwise.

Reply:
Thanks! Change applied.